src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -326,6 +328,12 @@ export type Visualizer = {
   readonly overrideProperty: (property: any) => void;
 };
 
+export type Viewport = {
+  readonly width?: number;
+  readonly height?: number;
+  readonly isMobile?: boolean;
```

Comment:
```suggestion
  readonly width: number;
  readonly height: number;
  readonly isMobile: boolean;
```

They should be always present

Reply:
Updated.